docs/export-functionality.md

```diff
@@ -4,17 +4,28 @@
 
 The Pricing module includes Export functionality, which allows the user to export data using different export criteria. Two types of data objects are allowed to be exported from Pricing module:
```

Comment:
Need to replace Two types (e.g. for Following types, to reduce possible changes in case of other type would be added/removed).